### PR TITLE
Schema to support CMT field mapping and additional objects

### PR DESCRIPTION
**Pull request by callen-sfdo**
# Critical Changes

# Changes

# Issues Closed

# New Metadata
- Adding 4 new custom metadata types to support field mapping and additional objects
- Adding fields to Data Import object to map to new GAU Allocation and Opportunity contact Role data import object mappings.
# Deleted Metadata

### CODE REVIEW

File: src/package.xml [hunk 1]
```diff
@@ -869,7 +869,16 @@
         <members>DataImport__c.Donation_Stage__c</members>
         <members>DataImport__c.Donation_Type__c</members>
         <members>DataImport__c.FailureInformation__c</members>
-        <members>DataImport__c.HomeAddressImportStatus__c</members>
```
Comment (ptewson-sfdo) [inline src/package.xml]: My build failed without this line in package

**Comment by callen-sfdo** [comment]
**lurch attach: W-032130

**Comment by callen-sfdo** [comment]
**lurch attach: W-032295

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008LlvRQAS" target="_blank">W-032130</a>

**Comment by callen-sfdo** [comment]
**lurch: attach W-032130

**Comment by callen-sfdo** [comment]
**lurch: attach W-032295

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008LlvRQAS" target="_blank">W-032130</a>

**Pull request merged**